src/User_Command.php

```diff
@@ -692,26 +692,34 @@ public function set_role( $args, $assoc_args ) {
 	 * <user>
 	 * : User ID, user email, or user login.
 	 *
-	 * <role>
-	 * : Add the specified role to the user.
+	 * [<role>...]
+	 * : Add the specified role(s) to the user.
 	 *
 	 * ## EXAMPLES
 	 *
 	 *     $ wp user add-role 12 author
 	 *     Success: Added 'author' role for johndoe (12).
 	 *
+	 *     $ wp user add-role 12 author editor
+	 *     Success: Added 'author' role for johndoe (12).
+	 *     Success: Added 'editor' role for johndoe (12).
+	 *
 	 * @subcommand add-role
 	 */
 	public function add_role( $args, $assoc_args ) {
 		$user = $this->fetcher->get_check( $args[0] );
 
-		$role = $args[1];
-
-		self::validate_role( $role );
+		$roles = $args;
+		array_shift( $roles );
```

Comment:
Can we error early here if `$roles` is empty, and then make sure we have a test for the scenario too?

Reply:
@danielbachhuber Yep, I'll work on this now.